### PR TITLE
Configuring repository to use Antora

### PR DESCRIPTION
**Pull request by gastaldi**
To be displayed in https://quarkiverse.github.io/quarkiverse-docs/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ hello.process(Map.of("name", "bob"), stringWriter);
 String result = stringWriter.toString();
 ```
 
-For more details, check the complete [documentation](doc/freemarker.adoc).
+For more details, check the complete [documentation](docs/modules/ROOT/pages/index.adoc).
```
Comment (gsmet) [inline README.md:36]: Is it really necessary to put the doc so deep in a hierarchy? Doesn't look very developer friendly?

Comment (gastaldi) [reply README.md:36]: Yes, this is the Antora's Standard File and Directory layout.

See https://docs.antora.org/antora/2.3/standard-directories/#hierarchy-and-reserved-names

Comment (gastaldi) [reply README.md:36]: Actually this link is wrong, let me fix it

**Comment by vsevel** [comment]
looks good! 👍

**Pull request merged**